diagnostic.md

```diff
@@ -5,74 +5,90 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+A backend's purpose is to handle and respond to requests from the client, or front end.  It handles things like business logic and database interactions.  The back end essentially enables the front end experience.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Views are the presentation layer and are essentially visual representations of models.  They render the models into one or more formats, such as XML.
```

Comment:
Yes but why don't we use views on the back-end? The answer is because we do it on the front-end to minimize requests to the server.